src/utils/eth.ts

```diff
@@ -0,0 +1,17 @@
+import { providers } from 'ethers'
+import { ChainId } from '@dcl/schemas'
+import { RPC_URLS } from 'decentraland-connect'
+import { env } from 'decentraland-commons'
+import { isContract } from 'decentraland-transactions'
+
+export const MATIC_CHAIN_ID = env.get(
+  'MATIC_CHAIN_ID',
+  ChainId.MATIC_MAINNET
+) as ChainId
+
+export async function isPublished(collectionAddress: string) {
+  const rpcUrl = RPC_URLS[MATIC_CHAIN_ID]
+  const provider = new providers.JsonRpcProvider(rpcUrl)
```

Comment:
are there any benefits to having this as a local variable instead of re-instantiating on each request?

Reply:
nope, i moved it outside 👍